headers/types/types.h

```diff
@@ -5,5 +5,6 @@
 #include "dungeon_mode/dungeon_mode.h"
 #include "ground_mode/ground_mode.h"
 #include "sentry_duty/sentry_duty.h"
+#include "sound/sound.h"
```

Comment:
Can we put this in `common/sound.h`? The sound engine isn't its own mode.

Reply:
Done.